keystrokes/src/DB/dbTestData.json

```diff
@@ -831,11 +821,14 @@
             "race": "NIGHT_ELF",
             "name": "RiftWood",
             "specs": {
-                "Restoration": {
+                "RESTORATION": {
                     "spec": "RESTORATION",
                     "enabled": true,
                     "configured": true,
-                    "keyBindings": "1"
+                    "keyBindings": {
+                        "1": 1,
```

Comment:
What's the purpose here?

Reply:
it was for a mapping, i decided that there could be multiple keybindings per spec, so i needed to change that. think it could probably be {0: "id_for_keybindings", 1: "id_for_other_keybindings"}.